packages/eslint-plugin/docs/rules/restrict-template-expressions.mdx

```diff
@@ -65,8 +67,13 @@ const msg2 = `arg = ${arg || 'zero'}`;
 
 This option controls both numbers and BigInts.
 
+We recommend avoiding using this option if you use any floating point numbers.
```

Comment:
```suggestion
We recommend using this option if you use any floating point numbers.
```

Reply:
Err. The meaning right now is what I'd wanted to convey: if you use any floating point numbers, don't use this option. No?

Reply:
Ah, I get it. Yes.